frontend/package.json

```diff
@@ -70,8 +70,10 @@
     "lint:stylelint:write": "stylelint '**/*.scss' --fix",
     "test": "react-scripts test",
     "test:ci": "CI=true yarn test --coverage",
-    "e2e": "cd cypress && docker-compose up -d && docker-compose logs -f cypress && docker-compose down",
```

Comment:
Same comment as above - why are we not using Docker here?

Reply:
@jdorothy for context, I spent about a week setting up a docker compose setup for the e2e tests. Alis was the first person to try and run it on their machine, and kept running into various issues getting it working. We paired trying to get it to work over several hourlong sessions with no luck. We decided it wasn't worth the time it was sinking - and when we switched to running locally, Alis had passing tests in under 30 minutes. So tbh I'm kinda just over Docker at this point - it's supposed to be a time saving tool but it's not really panning out that way.

Reply:
Unfortunately I can second @nickclyde's experience here - in spite of his excellent & detailed instructions on running the Dockerized e2e setup, there was plenty of heartburn for me en route to a dead end.

Reply:
After a bit of a painful grind, I've made docker-compose work and updated everything to use it.